cdap-api/src/main/java/co/cask/cdap/api/data/schema/SchemaTypeAdapter.java

```diff
@@ -14,9 +14,9 @@
  * the License.
  */
 
-package co.cask.cdap.internal.io;
+package co.cask.cdap.api.data.schema;
```

Comment:
Meaning just move the class from `cdap-common` to `cdap-api` but keeping the package the same 
https://github.com/caskdata/cdap/tree/develop/cdap-api/src/main/java/co/cask/cdap/internal/io